README.md

```diff
@@ -2,39 +2,45 @@
 This utility patches the official AM2R 1.1 release (Windows) to the fan-made Community Update (Linux).
 
 ## Dependencies
-The patcher and the installer only require a small amount of dependencies:
+The patcher and the installer only require a small amount of dependencies. Note that the Java-related packages (`java`, `jre`, `jdk`) packages are only required for building an Android APK.
 
 ### Arch (including Manjaro, EndeavourOS, RebornOS, etc.)
-Make sure that multilib is enabled, as `lib32-libpulse` is a 32-bit library, and Arch does not enable 32-bit support by default.  
+Make sure that multilib is enabled, as `lib32-libpulse` is a 32-bit library, and Arch does not enable 32-bit support by default.
 To enable it, go to `/etc/pacman.conf`, search for `[multilib]`, and make sure that the next two lines are uncommented:
 ```
 [multilib]
 Include = /etc/pacman.d/mirrorlist
 ```
 
 Then install the following dependencies:  
-`sudo pacman -S --needed unzip sed xdelta3 jre8-openjdk lib32-libpulse`
+`sudo pacman -S --needed unzip sed patchelf xdelta3 jre8-openjdk lib32-libpulse`
 
 
 ### Debian (including Ubuntu, Mint, PopOS, etc.)
-Make sure that the i386 architecture is enabled for you, as libopenal1 requires the 32-bit version, and Ubuntu does not enable 32-bit support by default.  
+Make sure that the i386 architecture is enabled for you, as libopenal1 requires the 32-bit version, and Ubuntu does not enable 32-bit support by default.
 To enable it, do the following:
 ```
 sudo dpkg --add-architecture i386
-sudo apt update && sudo apt install libc6:i386 libncurses5:i386 libstdc++6:i386
+sudo apt update && sudo apt install libc6:i386 libncurses5:i386 libstdc++6:i386 libopenal1:i386
 ```
 
 Then install the following dependencies:  
-`sudo apt install unzip sed xdelta3 openjdk-8-jre libopenal1:i386`
+`sudo apt install unzip sed xdelta3 patchelf openjdk-8-jre`
+
+### Nix (Including NixOS)
+Enter a shell with all the required dependencies to run the script:
+`nix shell nixpkgs#{gnused,unzip,xdelta,patchelf,jre}`
+
+Note that the AppImage will **not** work on NixOS, please follow the instructions in the manual installation section.
 
 ### Red Hat (including Fedora)
-`sudo yum unzip sed xdelta java-1.8.0-openjdk`
+`sudo yum unzip sed xdelta patchelf java-1.8.0-openjdk`
 
 ### Suse
-`sudo zypper unzip sed xdelta3 java-1_8_0-openjdk`
+`sudo zypper unzip sed xdelta3 patchelf java-1_8_0-openjdk`
 
 ## Controllers
-In order to be able to use a controller, you *need* the two following packages as well:
+On some distributions, in order to be able to use a controller, you *need* the two following packages as well:
```

Comment:
Should be "In order to use a controller on some distributions, the two following packages are *mandatory*:"